reference/7.0/Microsoft.PowerShell.Core/About/about_Calculated_Properties.md

```diff
@@ -0,0 +1,506 @@
+that describe the property. The attributes define the name of the property, an
+expression to calculate the value, and optional formatting information. A
+calculated property is defined using a hashtable containing these attributes as
+key-value pairs.
+
+## Supported cmdlets
+
+The following cmdlets support calculated properties using the listed property
+attributes names as keys in the hashtable.
+
+> [!NOTE]
+> The `expression` attribute can be a script block instead of a hashtable. For
+> more information, see the [Notes](#notes) section. For descriptions of these
+> attibutes, see [Attribute definitions](#attribute-definitions).
+
+- `Compare-Object`
+  - `expression`
+
+- `ConvertTo-Html`
+  - `name`/`label` - optional
+  - `expression`
+  - `width` - optional
+  - `alignment` - optional
+
+- `Format-Custom`
+  - `expression`
+  - `depth` - optional
+
+- `Format-List`
+  - `name`/`label` - optional
+  - `expression`
+  - `formatstring` - optional
+
+  This same set of attributes also apply to the `-GroupBy` properties for all
+  `Format-*` cmdlets.
+
+- `Format-Table`
+  - `name`/`label` - optional
+  - `expression`
+  - `formatstring` - optional
+  - `width` - optional
+  - `alignment` - optional
+
+- `Format-Wide`
+  - `expression`
+  - `formatstring` - optional
+
+- `Group-Object`
+  - `expression`
+
+- `Measure-Object`
+  - Only supports a script block for the expression, not a hashtable.
+
+- `Select-Object`
+  - `name`/`label` - optional
+  - `expression`
+
+- `Sort-Object`
+  - `expression`
+  - `ascending`/`descending` - optional
+
+For more information, see the [Examples](#examples) section of this article.
+
+## Attribute definitions
+
+- `name`/`label` - Specifies the name of the property being created. You can
+  use `name` or its alias, `label`, interchangeably.
+- `expression` - A script block used to calculate the value of the new property.
+- `alignment` - Used by cmdlets that produce tabular output to define how the
+  values is displayed in a column. The value must be `'left'`, `'center'`, or
+  `'right'`.
+- `formatstring` - Specifies a format string that defines how the value is
+  formatted for output. For more information about format strings, see
+  [Format types in .NET](/dotnet/standard/base-types/formatting-types).
+- `width` - Specifies the maximum width column in a table when the value is
+  displayed. The value must be greater than `0`.
+- `depth` - The **Depth** parameter of `Format-Custom` specifies the depth of
+  expansion for all properties. The `depth` attribute allows you to specify the
+  depth of expansion per property.
+- `ascending` / `descending` - Allows you to specify the order of sorting for
+  one or more properties. These are boolean values.
+
+The hashtable keys need not be spelled out as long as the specified name prefix
+is unambiguous. For example, `n` can be used in lieu of `Name` and `e` can be
+used in lieu of `Expression`.
```

Comment:
could a negative example be shown to highlight the concept in both forms?

Reply:
The examples use a variety of forms.